docs/secpoll.zone

```diff
@@ -516,4 +516,5 @@ dnsdist-1.8.0-rc2.security-status                          60 IN TXT "2 Unsuppor
 dnsdist-1.8.0-rc3.security-status                          60 IN TXT "2 Unsupported pre-release (no known vulnerabilities)"
 dnsdist-1.8.0.security-status                              60 IN TXT "1 OK"
 dnsdist-1.8.1.security-status                              60 IN TXT "1 OK"
-dnsdist-1.9.0-alpha1.security-status                       60 IN TXT "1 Unsupported pre-release (no known vulnerabilities)"
+dnsdist-1.9.0-alpha1.security-status                       60 IN TXT "2 Unsupported pre-release (no known vulnerabilities)"
```

Comment:
I know conflict resolution will remind you as well, but alpha1 of course has the http/2 issue

Reply:
Fixed, thanks!